### PR TITLE
feat(connect): update stellar-sdk and stellar-base

### PR DESCRIPTION
**Pull request by quietbits**
Updating `stellar-sdk` and `stellar-base` to the latest version to address breaking changes. `stellar-sdk` is exported as ESM instead of CJS now.

### CODE REVIEW

File: packages/connect-plugin-stellar/package.json [hunk 1]
```diff
@@ -24,13 +24,13 @@
...
         "@trezor/connect": "9.x.x",
-        "stellar-sdk": "^10.4.1"
+        "stellar-sdk": "^11.0.0-beta.3"
     },
     "devDependencies": {
         "jest": "^26.6.3",
         "rimraf": "^5.0.1",
-        "stellar-base": "^8.2.2",
-        "stellar-sdk": "^10.4.1",
+        "stellar-base": "^10.0.0-beta.1",
```
Comment (Shaptic) [inline packages/connect-plugin-stellar/package.json:32]: I just released a beta.2 if you want to use that here; it shouldn’t make a material difference, though

**Comment by mroz22** [comment]
ciao, please run `yarn dedupe` and commit changes.

**Comment by mroz22** [comment]
now `yarn build:libs` step is failing. https://github.com/trezor/trezor-suite/actions/runs/6291766117/job/17088806725?pr=9543#step:13:147

I am not really sure what in your PR might be causing this. Maybe you got some idea? If not, I would try rebasing it on latest develop and trying again..

**Comment by quietbits** [comment]
@mroz22 Pulling in the latest `develop` didn't fix this. There must be something with our libraries, because `develop` `build:libs` works. I only checked Stellar package builds, sorry about that! We'll try to fix the issue. :crossed_fingers:

**Comment by Shaptic** [comment]
@mroz22 in one of the updates to the Stellar SDK (specifically, [`v11.0.0-beta.1`](https://github.com/stellar/js-stellar-sdk/releases/tag/v11.0.0-beta.1)), we tightened up our dependencies for modern deployment environments. This included dropping [`es6-promise`](https://www.npmjs.com/package/es6-promise), which I _think_ you guys were actually relying on to polyfill promises for older browsers in other packages, but never explicitly added as a dependency.

In other words, I think the Stellar connection was transiently/"accidentally" providing that functionality in the dependency chain for everyone else. But, since we don't need it and you do, it needs to be explicitly added to `packages/connect/package.json`, now.

I know this seems odd, but I can't otherwise explain how running `yarn add es6-promise` in `/packages/connect` (on top of this branch) makes all tests pass. Does that seem possible?

**Comment by mroz22** [comment]
thank you @Shaptic you are right. fix is incoming https://github.com/trezor/trezor-suite/pull/9589

**Comment by mroz22** [comment]
its now merged into develop, please rebase
